Add Hyperkube removal warning

evrardjp opened the pull request:
We should warn our customers in advance that we'll drop the hyperkube
image. This should do it. During the 4.3.0 release, we should ensure
another release note clarifies what are the new image names.

Commit:
Add Hyperkube removal warning

We should warn our customers in advance that we'll drop the hyperkube
image. This should do it. During the 4.3.0 release, we should ensure
another release note clarifies what are the new image names.

ereslibre reviewed (approved):
Thank you @evrardjp, LGTM.

kkaempf commented on adoc/MAIN.release-notes.adoc:
```diff
@@ -19,6 +19,12 @@ Release notes usually only list changes that happened between two subsequent rel
+== Deprecations in 4.2.x
+
+- The hyperkube image, combining multiple kubernetes binaries, is planned for
+  removal in 4.3.0. If running {productname} in an airgapped environment,
```
Let's make it extra-clear that this is not a random removal orginating from SUSE:

"...removal in 4.3.0 *due to upstream deprecations*."

evrardjp replied:
good idea.

Commit:
Clarify the hyperkube deprecation reason

It is not clear why we are deprecating the hyperkube image.
This should clarify that it is done to sync with upstream.

jordimassaguerpla commented:
merged in https://github.com/SUSE/caasp-release-notes/pull/47
